src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -67,8 +70,22 @@ Tree SteinerTreeBuilder::makeSteinerTree(odb::dbNet* net,
                                          std::vector<int>& y,
                                          int drvr_index)
 {
-  float net_alpha = net_alpha_map_.find(net) != net_alpha_map_.end() ?
-                    net_alpha_map_[net] : alpha_;
+  float net_alpha = alpha_;
```

Comment:
Why is this done here and not in the common code (ie makeTree)?  The other makeSteinerTree won't obey these settings.

Reply:
The other makeSteinerTree doesn't have `dbNet` information and it's only used by TR right now. The `makeTree` function also doesn't have the `dbNet` info. I didn't add these checks in there because at some point, this two functions will be removed (after TR update)

Reply:
I think you should just use the hpwl of the given x/y since that is what you are actually building the tree of.  The net is really only used for alpha lookup.

Reply:
It will not work. FastRoute doesn't pass the real x/y values, but the grid positions. The hpwl computed in this case will not match the actual hpwl of the net.